### PR TITLE
Fix a small typo

### PR DESCRIPTION
**Pull request by BassemN**

### CODE REVIEW

**Comment by inoas** [comment]
LGTM.

**Comment by bravo-kernel** [comment]
Nice 🎣

File: config/bootstrap.php [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 // You can remove this if you are confident that your PHP version is sufficient.
 if (version_compare(PHP_VERSION, '5.5.9') < 0) {
-    trigger_error('You PHP version must be equal or higher than 5.5.9 to use CakePHP.', E_USER_ERROR);
+    trigger_error('Your PHP version must be equal or higher than 5.5.9 to use CakePHP.', E_USER_ERROR);
```
Comment (dereuromark) [inline config/bootstrap.php]: Ideally, the version would be variableized :)

Comment (inoas) [inline config/bootstrap.php]: That is true, however we should do that as a global possibly. One that gets defined in index.php?

Comment (dereuromark) [inline config/bootstrap.php]: Inside each file at least one version would also be a good start probably.

Comment (inoas) [inline config/bootstrap.php]: A lot of boilerplate. Why not `define('CAKEPHP_PHP_MIN_VERSION', '5.5.9')`;
Another way would be to read it out from composer.json within index.php/bootstrap?
The book also references the minimum version, as do some tool setup files. Would be great to have those things centralised somewhere.

**Pull request merged**